uberfire-nio2-backport/uberfire-nio2-api/src/test/java/org/uberfire/java/nio/file/FilesTest.java

```diff
@@ -528,55 +513,42 @@ public void moveFileInvalidSourceAndTarget() throws IOException {
         final Path dest = newTempDir().resolve("other");
 
         final Path sourceFile = source.resolve("file.txt");
-        final OutputStream stream = Files.newOutputStream(sourceFile);
-        stream.write('a');
-        stream.close();
-
-        try {
-            Files.move(source,
-                       dest);
-            fail("source isn't empty");
-        } catch (Exception ex) {
+        try (final OutputStream stream = Files.newOutputStream(sourceFile)) {
+            stream.write('a');
         }
 
+        assertThatThrownBy(() -> Files.move(source, dest))
+                .isInstanceOf(DirectoryNotEmptyException.class);
+
         sourceFile.toFile().delete();
         Files.copy(source,
                    dest);
 
-        try {
-            Files.move(source,
-                       dest);
-            fail("dest already exists");
-        } catch (Exception ex) {
-        }
+        assertThatThrownBy(() -> Files.move(source, dest))
+                .isInstanceOf(FileAlreadyExistsException.class);
 
         dest.toFile().delete();
         source.toFile().delete();
 
-        try {
-            Files.move(source,
-                       dest);
-            fail("source doesn't exists");
-        } catch (Exception ex) {
-
-        } finally {
-        }
+        assertThatThrownBy(() -> Files.move(source, dest))
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessage("Condition 'source must exist' is invalid!");
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void moveNull1() throws IOException {
+    public void moveNull1() {
         Files.move(newTempDir(),
                    null);
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void moveNull2() throws IOException {
+    public void moveNull2() {
         Files.move(null,
                    newTempDir());
     }
 
     @Test(expected = IllegalArgumentException.class)
```

Comment:
why preserving here the "expected"? left over?

Reply:
Well spotted, I must have missed that. I'll replace the remaining occurrences tomorrow.